Assets/Editor/BuildPipeline/iOSTestPostProcessor.cs

```diff
@@ -0,0 +1,45 @@
+#if UNITY_EDITOR
+namespace Shopify.Unity.Editor.BuildPipeline {
+    using UnityEngine;
+    using UnityEditor;
+    using UnityEditor.iOS.Xcode;
+    using System.IO;
+    using System;
+
+    public class iOSTestPostProcessor {
```

Comment:
Since all these methods are static we can't really use base class calls so we don't gain much unless we move everything to non static and instantiate an instance in the build pipeline. Thoughts?